src/user-research/choosing-and-using-analysis-and-synthesis-methods.md

```diff
@@ -15,7 +15,13 @@ This guide lists our favourite books, articles and videos on the analysis and sy
 
 The purpose of analysis and synthesis (sometimes also called sensemaking) is to produce findings (sometimes also called insights) that can inform what our team, and sometimes the wider organisation, does next.
 
-We do at least an initial analysis and synthesis at the end of each ‘batch’ of research, so that we can quickly [learn, share and adapt](https://playbook.dxw.com/user-research/#user-research-principles). Later we may combine findings from different research activities into a coherent picture. And we plan the pace of our research to avoid building up analysis and synthesis debt.
+### Principles
+
+We do at least an initial analysis and synthesis at the end of each research activity, so that we can quickly [learn, share and adapt](https://playbook.dxw.com/user-research/#user-research-principles).
+
+Later we may combine different kinds of data and findings from different research activities to create a coherent picture.
+
+And we plan the pace of our research to avoid building up analysis and synthesis debt.
 
 Our rule of thumb is “one part research to two parts communication”. For a day spent doing interviews, for example, we expect to spend the next two days on analysis and synthesis, and sharing your findings.
```

Comment:
I would change the end of this sentence from "and sharing your findings." to "and sharing findings".

Reply:
I've started a new pull request with that, and other, improvements to the wording.